client/src/main/java/keywhiz/client/KeywhizClient.java

```diff
@@ -419,4 +429,12 @@ private String httpDelete(HttpUrl url) throws IOException {
 
     return makeCall(request);
   }
+
+  private String httpDeleteQuietly(HttpUrl url) {
+    try {
+      return httpDelete(url);
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+  }
```

Comment:
We're also bucking the trend of having every method throw IOException (which is checked and thus annoying to deal with) in favor of wrapping it.